### PR TITLE
Translate 6.24 to Chinese

### PR DESCRIPTION
**Pull request by blackmatch**
1. translate 6.24 to Chinse
2. fix "return to top" bug.

### CODE REVIEW

**Comment by goldbergyoni** [comment]
Yo @blackmatch , welcome and many thanks 🌻

I asked for @MattJin review since I just don't speak Chinese:)

File: sections/security/saferedirects.chinese.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# 避免不安全的重定向
+
+### 一段解释
+
+当我们在Node.js或者Express中实现重定向时，在服务器端进行输入校验非常重要。当攻击者发现你没有校验用户提供的外部输入时，他们会在论坛、社交媒体以和其他公共场合发布他们精心制作的链接来诱使用户点击，以此达到漏洞利用的目的。
```
Comment (bushuai) [inline sections/security/saferedirects.chinese.md:5]: ✨ 中英文之间可以加个空格，提升阅读体验哦。

```
// before
当我们在Node.js或者Express中实现重定向时...

// after
当我们在 Node.js 或者 Express中 实现重定向时...
```

Comment (blackmatch) [reply sections/security/saferedirects.chinese.md]: 感谢提醒，已修改。

Comment (goldbergyoni) [reply sections/security/saferedirects.chinese.md]: @blackmatch @bushuai LMK your final thoughts on this?

Comment (blackmatch) [reply sections/security/saferedirects.chinese.md]: @goldbergyoni This comment is talking about to make a better reading experience. And I did it at the last commit. I'm not sure if this PR needs more review. @bushuai what do you think?

Comment (bushuai) [reply sections/security/saferedirects.chinese.md]: Looks good! 😉

**Comment by Archangel-Ambriel** [comment]
Are you trying to play with me? Or are you at my side? Better be on my side
now when I really start to show you my skills and levels of real magic!

mån 13 juli 2020 kl. 04:05 skrev bu <notifications@github.com>:

**Review by MattJin — DISMISSED** [review]
looks good!

**Review by MattJin — APPROVED** [review]

**Pull request merged**